graphdatascience/resources/imdb/serialize_imdb.py

```diff
@@ -0,0 +1,68 @@
+from importlib.resources import path
+
+import pandas as pd
+from pandas import read_pickle
+
+"""
+This code included only for reproducibility.
+
+It is used to convert raw pickle data from edges.pkl, labels.pkl and node_features.pkl
+into consolidated imdb_nodes_gzip.pkl and imdb_rels_gzip.pkl.
+
+These pickles are then being read by load_imdb()
+"""
+
+with path("graphdatascience.resources.imdb", "raw/labels.pkl") as labels_resource:
+    class_labels = read_pickle(labels_resource)
+movies = pd.DataFrame([item for sublist in class_labels for item in sublist])
+movies = movies.rename(columns={0: "nodeId", 1: "class"})
```

Comment:
Might be call the property "genre" instead of "class" to give it some semantic meaning. And indeed, it may not be used for classification after all, in which "class" does not really make sense

Reply:
The same comment for "feature" below. I would instead call it "plot_keywords" or something similar